presto-cassandra/src/main/java/io/prestosql/plugin/cassandra/CassandraType.java

```diff
@@ -327,6 +327,7 @@ public String toCqlLiteral(Object prestoNativeValue)
             case ASCII:
             case TEXT:
             case VARCHAR:
+            case DATE:
```

Comment:
Is it OK to send day number to cassandra, or should we rather send a string in yyyy-mm-dd format?

Reply:
Thanks for the review. Cassandra expects quoted String in yyyy-MM-dd format